shared/chat/conversation/messages/react-button/index.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 import * as React from 'react'
+import * as ReactDOM from 'react-dom'
```

Comment:
I thought that this would break debugging on native ([docs](https://github.com/keybase/client/blob/44fb7e9740ce778c5c6d8286cd9c7ad494b25c8a/shared/docs/react-native/troubleshooting.md#expected-to-find-exactly-one-react-native-renderer-on-devtools-hook)), but I guess not? Maybe we should remove that section

Reply:
yeah we shouldn't do this